proof/verifier.go

```diff
@@ -356,6 +356,17 @@ func (p *Proof) verifyGroupKeyReveal() error {
 // block header is invalid (usually: not present on chain).
 type HeaderVerifier func(blockHeader wire.BlockHeader, blockHeight uint32) error
 
+// GroupVerifier is a callback function which returns an error if the given
+// group key has not been imported by the tapd daemon. This can occur if the
+// issuance proof for the group anchor has not been imported or synced.
+type GroupVerifier func(groupKey *btcec.PublicKey) error
```

Comment:
Do we just actually want a predicate here, or is the error useful down the line for test assertions?

Reply:
Error was useful while working on tests - I think also we want to pass any DB error that could occur inside the verifier vs. only handling the expected error of a missing group.